website/docs/r/servicebus_namespace.html.markdown

```diff
@@ -64,6 +66,19 @@ A `identity` block supports the following:
 
 * `identity_ids` - (Optional) A list of User Managed Identity ID's which should be assigned to the ServiceBus Namespace.
 
+---
+
+-> **Note:** Once customer-managed key encryption has been enabled, it cannot be disabled.
```

Comment:
should we have a custom diff that forces a new resource if the key is removed?

what happens if we change the key?

Reply:
Good idea! I've added the custom diff.

And it allows user to change the key and identity, but can't toggle `infrastructure_encryption_enabled`, so I added `ForceNew` to it.

Please take another look, thanks!